pontoon/insights/tasks.py

```diff
@@ -0,0 +1,391 @@
+import logging
+
+from celery import shared_task
+from datetime import timedelta
+
+from django.contrib.auth.models import User
+from django.utils import timezone
+
+from pontoon.actionlog.models import ActionLog
+from pontoon.base.models import Entity, Locale, Translation
+from pontoon.base.utils import group_dict_by
+from pontoon.insights.models import LocaleInsightsSnapshot
+
+
+log = logging.getLogger(__name__)
+
+
+@shared_task(bind=True)
+def collect_insights(self):
+    """
+    Gather data needed for the Insights tab and store it in the DB.
+    """
+    start_of_today = (
```

Comment:
[nitpick] I'm not sure about the types in the rest of the patch, but `timezone.now().date()` could produce the same results.

Reply:
I feel stupid.

Reply:
Oh wait, that results in the naive `datetime`. It should be timezone aware instead.

Reply:
I'm sorry, I thought that Django somehow handles that case.